File: app/src/main/java/com/losextraditables/krokodil/core/model/Video.java
package com.losextraditables.krokodil.core.model;

import java.util.Date;
import java.util.List;
import lombok.Data;

@Data
public class Video {
  private String id;
  private String duration;
  private Date publishedAt;
  private String title;
  private String description;
  private List<String> thumbnails;
}

File: app/src/main/java/com/losextraditables/krokodil/core/model/Video.kt
package com.losextraditables.krokodil.core.model

import java.util.*

data class Video(
        val id: String,
        val duration: String,
        val publishedAt: Date,
        val title: String,
        val description: String,
        val thumbnails: List<String>,
        val viewCount: Long,
        val channelTitle: String
)